### DISCUSSION TITLE
HAA with Sonoff doesn't connect to wifi

### DISCUSSION THREAD

**Discussion by claude5555** [RavenSystem/esp-homekit-devices#1771] [Q&A]
Hi,

I am a newbie. I upload the firmware and got the setup menu. I entered all detail of the wifi connection and the device doesn't connect to my wifi system. I always get back to the setup menu. 

May I please get some help to understand why ad what I have to do.

Thanks

**Comment by Kienz** [comment]
Do you have some sort of Mac filtering active? Is the password 100% correct - no whitespaces before/after? How is the signal strength?

**Comment by claude5555** [comment]
No I don’t and the password is 100%.

Do I have to connect to Bssid or Ssid.

I connected to ssid, but my network has 3 bssid.

We agree that I have to upload HAAfull.bin ?

Does it work with sonoff basic?

Thank you for your help and kindness

Le 4 mars 2022 à 08:47, Stefan Kienzle ***@***.***> a écrit :

﻿

Do you have some sort of Mac filtering active? Is the password 100% correct - no whitespaces before/after? How is the signal strength?

—
Reply to this email directly, view it on GitHub<https://github.com/RavenSystem/esp-homekit-devices/discussions/1771#discussioncomment-2293386>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AENYBHHK5EMTMEYNG6BNM4TU6G5X5ANCNFSM5P4Y3SBA>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

**Comment by Kienz** [comment]
I think it should work.
You have to flash fullhaaboot.bin - https://github.com/RavenSystem/esp-homekit-devices/wiki/Installation#installing-haa

SSID and BSSID should work.
Can you post your JSON config? For debugging please add "o":3 into general config "c".
https://github.com/RavenSystem/esp-homekit-devices/wiki/General-Configuration#log-output
After that open console and run the following command ```nc -kulnw0 45678``` to get logs.

**Comment by claude5555** [comment]
Thank a lot Stephan. I am not at home at the moment. I will try this end of afternoon and I will let you know.

Claude

Le 4 mars 2022 à 10:00, Stefan Kienzle ***@***.***> a écrit :

﻿

I think it should work.
You have to flash fullhaaboot.bin - https://github.com/RavenSystem/esp-homekit-devices/wiki/Installation#installing-haa

SSID and BSSID should work.
Can you post your JSON config? For debugging please add "o":3 into general config "c".
https://github.com/RavenSystem/esp-homekit-devices/wiki/General-Configuration#log-output
After that open console and run the following command nc -kulnw0 45678 to get logs.

—
Reply to this email directly, view it on GitHub<https://github.com/RavenSystem/esp-homekit-devices/discussions/1771#discussioncomment-2293812>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AENYBHBN4Z6SKP5YWVJ4AXDU6HGJNANCNFSM5P4Y3SBA>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

**Comment by claude5555** [comment]
Hi

I have another question. Do I have to populate bssid or can I keep it empty?

Thanks

Claude

Le 4 mars 2022 à 10:00, Stefan Kienzle ***@***.***> a écrit :

﻿

I think it should work.
You have to flash fullhaaboot.bin - https://github.com/RavenSystem/esp-homekit-devices/wiki/Installation#installing-haa

SSID and BSSID should work.
Can you post your JSON config? For debugging please add "o":3 into general config "c".
https://github.com/RavenSystem/esp-homekit-devices/wiki/General-Configuration#log-output
After that open console and run the following command nc -kulnw0 45678 to get logs.

—
Reply to this email directly, view it on GitHub<https://github.com/RavenSystem/esp-homekit-devices/discussions/1771#discussioncomment-2293812>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AENYBHBN4Z6SKP5YWVJ4AXDU6HGJNANCNFSM5P4Y3SBA>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

**Comment by Kienz** [comment]
It can be empty. You just select Wifi SSID and enter password.

**Comment by claude5555** [comment]
Hi,

Sorry for this late answer. I came back home later in the evening than I expected.

I followed your recommendations this morning and it works. I think that the issue came from the json I added that included 3 buttons instead of only one ( so off basic). I cancelled the others buttons.

Now I gonna try to add a sensor ( bme680), but, if I can ask more help from you, my goal now is to upload this firmware for lightbulb by ota.

Do you know how I can make?

I want to thank for your help and patience

Regards

Claude

Le 4 mars 2022 à 15:47, Stefan Kienzle ***@***.***> a écrit :

﻿

It can be empty. You just select Wifi SSID and enter password.

—
Reply to this email directly, view it on GitHub<https://github.com/RavenSystem/esp-homekit-devices/discussions/1771#discussioncomment-2295635>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AENYBHEEQR32LG25ODAUNCTU6IPAFANCNFSM5P4Y3SBA>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
